### PR TITLE
Added Engineering Management ep 1 in activities

### PR DESCRIPTION
**Pull request by MahdeeMushfiqueKamal**

### CODE REVIEW

File: _data/activitiesByCategory.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+- category:  Android Workshop
+  side_bar_name: Android Workshop
+
+  categoryDesc:
   sessionFolderPath: "_activities/AndroidWorkshop"
 
+- category: Engineering Management
+  side_bar_name: Engineering Management
+  categoryDesc:
+  sessionFolderPath: "_activities/EngineeringManagement"
```
Comment (imranhasanhira) [inline _data/activitiesByCategory.yml]: can we skip CamelCase in url paths ? I would prefer to use `engineering-management`

File: _activities/EngineeringManagement/overview.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+layout: sessionTemplate
+title: Overview 
+desc: >
+ Here goes description + written question ans  lkasjfdlk l asdklfljslkad fklsf jkllasdjfk safjdas fjsklf jkldfjlk
```
Comment (imranhasanhira) [inline _activities/EngineeringManagement/overview.md]: Write some real description here :)

File: _data/activitiesByCategory.yml (hunk 1 above)
Comment (MahdeeMushfiqueKamal) [reply _data/activitiesByCategory.yml]: URL path is fixed in the 2nd commit @imranhasanhira vaia

File: _activities/EngineeringManagement/overview.md (hunk 2 above)
Comment (MahdeeMushfiqueKamal) [reply _activities/EngineeringManagement/overview.md]: Added a description. @imranhasanhira vaia

**Review by imranhasanhira — APPROVED** [review]

**Pull request merged**